sparse_ho/models.py

```diff
@@ -226,16 +221,17 @@ def get_L(X, is_sparse=False):
             return norm(X, axis=0) ** 2 / (X.shape[0])
 
     def sk(self, X, y, alpha, tol, max_iter):
```

Comment:
this should not be a public method and should probably not be called sk. Next is to use celer here.